tests/project.bats

```diff
@@ -8,8 +8,11 @@ setup() {
 }
 
 @test "project init and update and info" {
-	cd "$TEMPDIR" || exit
+	cd "$TEMPDIR" || exit 1
 
+	"$DAGGER" project init ./ --name "github.com/foo/bar"
+	test -d ./cue.mod/pkg
+	test -d ./cue.mod/usr
```

Comment:
Why duplicate?

Reply:
Missd this when rebasing conflicts, thx for :eyes: